### PR TITLE
ensure submitted ballot is correct size and with non empty ElG Pairs

### PR DESCRIPTION
**Pull request by emduc**
Addresses issues #17  to ensure that when a vote is casted, it's non empty and it won't fail the shuffle

### CODE REVIEW

File: contracts/evoting/mod_test.go [hunk 1]
```diff
@@ -192,6 +192,56 @@ func TestCommand_CastVote(t *testing.T) {
 	dummyElection.Status = types.Open
 	jsElection, _ = json.Marshal(dummyElection)
 
+	_ = snap.Set(dummyElectionIdBuff, jsElection)
```
Comment (nkcr) [inline contracts/evoting/mod_test.go:195]: you should not ignore an error

Comment (nkcr) [reply contracts/evoting/mod_test.go:195]: same bellow

**Review by nkcr — APPROVED** [review]
Nice 👍 
Just a question: what happens if the ballot is not encrypted with the public key ? I guess it would just end with a garbage decrypted ballot, but until we are sure about that I would create a new issue for this specific case, before closing #17 .

**Comment by emduc** [comment]
> 

You're right, I added a ballot from a different public key in the scenario test and it the decryption seems to never end :(. 
I commented this ballot in the scenario so that the it still works until we fix the issue, should I uncomment it ? Remove it ?

**Review by nkcr — APPROVED** [review]
Thank you, works fine like that!

**Pull request merged**